app/middleware/bot.js

```diff
@@ -93,6 +93,11 @@ export default (store: Store) =>
       }
 
       if (puppet.url.startsWith('https://www.immobilienscout24.de/Suche')) {
+        if(!puppet.url.includes("sorting=2")) {
```

Comment:
I think this should work:
```
        if (!puppet.url.includes('sorting=2')) {
          store.dispatch(setBotMessage('Sortierung ändern...️'));
          setImmediate(async () => {
            await electronUtils.evaluate(
              "window.location = window.location + '&sorting=2'"
            );
          });
          await sleep(2000);
          store.dispatch(setBotMessage(null));
          await handlePuppetReady();
          return;
        }
```
...but it would need more testing.

Reply:
After leaving it running for a while, I am indifferent. It does work great, however sometimes the bot breaks with this after solving a captcha. For example in my case the emoji with text bubbles vanishes, after the captcha (but also not always) 

But altogether I would say this is a nice solution that works reliable enough